README.md

```diff
@@ -159,7 +159,7 @@ the binary verification command above.
 ## Building Node.js
 
 See [BUILDING.md](BUILDING.md) for instructions on how to build
-Node.js from source.
+Node.js from source along with a list of officially supported platforms.
```

Comment:
Nit: Maybe put a comma after `source` so that it's clearer that it's "how to build from source" and "a list of officially supported platforms" rather than "how to build from source" and "how to build a list of officially supported platforms"?

Although I'm tot sure that adding the comma doesn't then risk interpretation as "how to build Node.js from source, as long as you also have the list of officially supported platforms because you need that to build from source." :-D

Maybe the best thing is to split it into two sentences?:

> See [BUILDING.md](BUILDING.md) for instructions on how to build Node.js from source. The document also contains a list of officially supported platforms.